operators/tokenizer/bpe_tokenizer_model.hpp

```diff
@@ -44,6 +44,58 @@ class BpeModel {
     }
   }
 
+  OrtxStatus LoadPreTokenizer(const json& bpe_model) {
+    auto node_pre_tokenizer = bpe_model.find("pre_tokenizer");
+    if (node_pre_tokenizer == bpe_model.end() || node_pre_tokenizer->is_null()) {
+      return {};
+    }
+
+    auto iter_type = node_pre_tokenizer->find("type");
+    if (iter_type == node_pre_tokenizer->end() || iter_type->is_null()) {
+      return {};
+    }
+
+    if (iter_type->get<std::string>() != "Sequence") {
+      return {kOrtxErrorNotImplemented, "Unsupported pretokenizer type!"};
```

Comment:
something in the test seems off - i saw in `test_pp_api.py` you use the `"amd/AMD-OLMo-1B-SFT-DPO"` model, and if you look in the `tokenizer.json` for it, `pre_tokenizer` type is "ByteLevel", not "Sequence"; so it would fail here (but in the CI it is passing). So - we should add "ByteLevel" to the supported types as well here, but also first identify why it is not failing the test currently, perhaps the type is not being extracted right.